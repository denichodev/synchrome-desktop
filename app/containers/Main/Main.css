.mainContainer {
  padding: 20px;
  flex: 1;  
  height: 1px;
}

.appBody {
  display: flex;
  flex-direction: column;
  overflow: auto;
}
