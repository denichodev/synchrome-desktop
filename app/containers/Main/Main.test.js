import React from 'react';
import Main from './Main';

function setup() {
  const props = {};

  const enzymeWrapper = mount(<Main {...props} />);

  return {
    props,
    enzymeWrapper
  };
}

describe('App', () => {
  it('should match snapshot', () => {
    const { enzymeWrapper } = setup();
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
