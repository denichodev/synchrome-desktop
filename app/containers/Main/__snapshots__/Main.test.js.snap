// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main app page should match its snapshot 1`] = `
<div
  className="app"
>
  <Header />
  <div
    className="app-body appBody"
  >
    <Sidebar />
    <main
      className="main mainContainer"
    >
      <Component />
    </main>
  </div>
  <Footer />
</div>
`;
